.github/workflows/config/spelling_allowlist.txt

```diff
@@ -135,6 +136,7 @@ instantiation
 instantiations
 intrinsics
 iterable
+jit
```

Comment:
Just curious which changes were requiring this? If it's textual documentation, I suspect we should be using `JIT` instead of `jit` (and therefore not make this change to allowlist.txt). And if `jit` is in the code, it seems like the spell-checker shouldn't be checking the code, so I wouldn't expect it to trip it up.

Reply:
I think it was one of the comments in the 'examples', see `docs/sphinx/examples/python_mlir/bernstein_vazirani.py` which use the newly defined decorator as - `@cudaq.kernel # or can pass (jit=True)`.

Reply:
In our C++ Doxygen docs, we surround the parameters (and other code) with backticks, and that also makes the spell checker realize that it is a code and not generic text that needs spell checking. Would doing that be reasonable?

Reply:
Addressed in 3b8532f